receiver/apachereceiver/metadata.yaml

```diff
@@ -76,6 +76,129 @@ metrics:
       monotonic: true
       aggregation: cumulative
     attributes: [ server_name ]
+  apache.bytes_per_request:
+    enabled: true
+    description: The average number of bytes per request.
```

Comment:
It's the number of bytes divided by value of `Total Accesses` which is the number of requests since startup.
Ref: https://github.com/apache/httpd/blob/db3bb198612fbd43438db0232d9b635eae9d3d63/modules/generators/mod_status.c#L503-L504